examples/pytorch/rgcn/model.py

```diff
@@ -76,6 +78,7 @@ class RelGraphEmbedLayer(nn.Module):
     """
     def __init__(self,
                  dev_id,
+                 out_dev_id,
```

Comment:
I think we need some comments about what is dev_id and what is out_dev_id.

Reply:
I expanded the comments for parameters `dev_id` and `out_dev_id`, and I also changed the name of `dev_id` to `storage_dev_id` to make it more clear it it's used for, in 
50a9b26.